source/_components/toon.markdown

```diff
@@ -21,52 +21,52 @@ redirect_from:
   - /components/switch.toon/
 ---
 
-The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat and sensors for power and gas consumption. The component also auto-detects any smart plugs, solar panels and smoke detectors connected to Toon and adds sensors and switches for them.
+The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat, sensors for power and gas consumption, sensors for solar production and several binary sensors for things like boiler burner on/off, hot tap water and boiler health status.
```

Comment:
```suggestion
The `toon` component platform can be used to control your Toon thermostat. This component adds a climate device for your Toon thermostat, sensors for power and gas consumption, sensors for solar production, several binary sensors for things like boiler burner on/off, hot tap water and boiler health status.
```

Reply:
Incorrect. I agree it feels like it should be a `,`, nevertheless, the and delimits to different sum up sections.